src/styles/watch.scss

```
@@ -0,0 +1,63 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  height: $watchSize;
+  width: $watchSize;
+  
+  margin: 0 auto;
+  padding: 5px;
+
+  border: 4px solid #062550;
+  border-radius: 50%;
+  background-color: #c396aa;
+
+  &__minute-arrow {
+    position: absolute;
+    height: $arrowHeight;
+    width: $arrowWidth;
+
+    background-color: #451a49;
+    border-radius: 8px;
+
+    transform-origin: bottom;
+    animation-name: move;
+    animation-duration: 3660s;
+    animation-timing-function: linear;
```

It can save you several lines in your code. :)
```suggestion
    animation: move 3660s linear;
```